internal/framework/service/rulesets/schema.go

```diff
@@ -90,7 +90,6 @@ func (r *RulesetResource) Schema(ctx context.Context, req resource.SchemaRequest
 				Required: true,
 				Validators: []validator.String{
 					stringvalidator.OneOfCaseInsensitive(cfv1.RulesetPhaseValues()...),
```

Comment:
we'll need to remove the value from https://github.com/cloudflare/cloudflare-go/blob/master/rulesets.go#L38 as that is where the available values come from today.